docs/development.md

```diff
@@ -146,14 +150,44 @@ This operand deploys the `template-validator` found in
 See the old `template-validator` [repository](https://github.com/kubevirt/kubevirt-template-validator)
 for more docs.
 
-#### `vm-console-proxy` operand
+#### `vm-console-proxy`
+
+Installs the VM console proxy found in `data/vm-console-proxy-bundle`.
+A pull request (PR) for updating the bundle is automatically created when a new version
+of the [vm-console-proxy](https://github.com/kubevirt/vm-console-proxy) is released.
+
+#### `tekton-pipelines`
+
+Installs the Tekton Pipelines found in `data/tekton-pipelines`.
+The bundle should be updated manually when a new version of the
```

Comment:
This is not accurate. Pipelines does not have to be updated manually when new tasks are released. 
Pipelines are referencing tasks only by name. So the pipelines have to be changed if task's name is changed or there has to be some functionality change in pipeline. Otherwise pipelines do not have to be updated

Reply:
Okay, I've changed it to:
```
Pipelines eliminate the need for manual updates when new tasks are added. They simply refer to tasks by their names. So, pipelines only require changes if a task's name is modified or if there are functional adjustments, but otherwise, they don't need to be updated.
```